utils/prezly/context.ts

```diff
@@ -0,0 +1,13 @@
+import { Category, Newsroom } from '@prezly/sdk/dist/types';
```

Comment:
Context is not a util. I think it deserves to have it's own directory `/contexts` 🙂

Reply:
good idea!

Reply:
should I also create a `hooks/` folder?

Reply:
As you wish. I'd vote for it 🙂

Reply:
yep, already did it, looks cleaner 👌🏻